Encounter Me/Pages/About.razor

```diff
@@ -0,0 +1,5 @@
+﻿@page "/about"
+
+<h1 class="page-title">About Us</h1>
+<br />
+<h3>Encounter Me - trail and stuff..</h3>
```

Comment:
I think it is not very informative. I suggest writing in at least what we have in Readme.md file